src/main/java/org/eclipse/uprotocol/transport/UTransport.java

```diff
@@ -35,28 +35,12 @@
 
 public interface UTransport {
 
-    /**
-     * Send a message (in parts) over the transport.
-     * @param source The source address for the message, (ex. publish topic, the return address 
-     * for rpc-request, or the rpc method for rpc response
-     * @param payload Actual message payload.
-     * @param attributes uProtocol header attributes.
-     * @return Returns {@link UStatus} with {@link UCode} set to the status code (successful or failure).
-     */
-    UStatus send(UUri source, UPayload payload, UAttributes attributes);
```

Comment:
I believe we should keep both - where default implementation can simply create a UMessage from the parts and call the other method.
Then we leave room for specific transports to optimize

Remember, UTransport is low level. very low level - if we don't optimize here our performance and throughput will not be very good and our developers will not be happy